client/src/App.js

```diff
@@ -1,29 +1,53 @@
-import React, { Component } from 'react';
+import React, { Component } from "react"
+import { Route } from "react-router-dom"
 
-import SavedList from './Movies/SavedList';
-import MovieList from './Movies/MovieList';
-import Movie from './Movies/Movie';
+import SavedList from "./Movies/SavedList"
+import MovieList from "./Movies/MovieList"
+import Movie from "./Movies/Movie"
 
 export default class App extends Component {
   constructor() {
-    super();
+    super()
     this.state = {
       savedList: []
-    };
+    }
   }
 
   addToSavedList = movie => {
-    const savedList = this.state.savedList;
-    savedList.push(movie);
-    this.setState({ savedList });
-  };
+    console.log(this.state.savedList)
+    console.log(movie)
+    this.state.savedList.filter(item => {
+      return item === movie
+    })
+    if (this.state.savedList.includes(movie)) {
+      this.setState(prevState => {
+        return { savedList: prevState.savedList }
+      })
+    } else {
+      let savedList = [...this.state.savedList, movie]
```

Comment:
Great job getting the saved list functionality. For an extra challenge, try preventing repeats in the saved list.